src/main/java/Duke.java

```diff
@@ -1,10 +1,323 @@
+import java.util.ArrayList;
+import java.io.IOException;
+import java.nio.file.FileAlreadyExistsException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.Scanner;
+import duke.task.*;
+import duke.exception.*;
+
 public class Duke {
+    static int taskCount = 0;
+    static ArrayList<Task> taskList = new ArrayList<Task>();
+    static final String COMMANDS = "Commands:\n    todo taskName\n    deadline deadlineName /by time\n" 
+            + "    event eventName /at time\n    list\n    done taskNumber\n    delete taskNumber\n    help\n    bye\n";
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
-    }
-}
+        displayWelcomeMessage();
+        loadHistory();
+        inputAndExecuteCommand();
+        saveHistory();
+        displayExitMessage();
+    }
+
+    private static void displayWelcomeMessage() {
+        String logo = "         __    _    _              ____        __           \n"
+                +"        / /_  (_)  (_)___ ___     / __ \\__  __/ /_____      \n"
+                +"       / __ \\/ /  / / __ `__ \\   / / / / / / / //_/ _ \\     \n"
+                +"      / / / / /  / / / / / / /  / /_/ / /_/ / ,< /  __/     \n"
+                +"     /_/ /_/_/  /_/_/ /_/ /_/  /_____/\\__,_/_/|_|\\___/     \n";
+        System.out.print(logo + "\n");
+        System.out.print("What do you have to do today?\n");
+        System.out.print(COMMANDS + "\n");
+    }
+
+    private static void loadHistory() {
+        String home = System.getProperty("user.dir");
+        createDirectory(home);
+        loadDataFile(home);
+    }
+
+    private static void createDirectory(String home) {
+        Path path = Paths.get(home, "data");
+        try {
+            Files.createDirectory(path);
+        } catch (FileAlreadyExistsException e) {
+            // do nothing
```

Comment:
If this is not doing anything, perhaps catching this exception is not needed. Is there another exception (besides this) that would be more useful?  Hint: Error messages could inform user better in certain cases.

Reply:
thank you! i will edit the file creation process to remove empty catch clauses